src/Microsoft.Azure.ServiceBus/ExceptionReceivedEventArgsAction.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace Microsoft.Azure.ServiceBus
+{
+    static class ExceptionReceivedEventArgsAction
```

Comment:
I assume you're asking me @nemakam. I'm on a fence about that. What would be the a strong use case for having it public?